integrated-models/Human-GEM/customSVG.tsv

```diff
@@ -0,0 +1 @@
+@Map name	filename
```

Comment:
Very good point! My view on the matter was that the `neo4j-data-generation` triggers warnings for missing files, including these `*SVG.tsv`, so I was thinking there were three options:
- ignore the warnings
- do not trigger warnings
- have placeholder files
Out of these three, having placeholder files seemed like the most reasonable option to me.
I must admit I did not consider the need of adding something in the `README`.

I am happy to address this however you see fit as part of this PR.

Reply:
Thanks for the explanation. The preference I have is not without tradeoffs. I think the solution is ok for now and I'm open to discussing more about this in the future.

Reply:
Intuitively it's not neat to have empty placeholder files, but I'm very fond of not having "false" warnings when building the stack. So either not trigger warnings for these files specifically or keeping them would be my suggestion, depends on how much time you think it's worth putting into this

Reply:
I'm then leaning towards taking this further as a separate issue #9

Reply:
I'm actually supporting the idea of having an empty placeholder file so that each model keeps the same data structure. Maybe one can add a comment line e.g.
`# At moment there is no custom SVG map for this model, however, it might be added in the future since we are constantly updating the model`